docs/spec/governance/state.md

```diff
@@ -2,47 +2,58 @@
 
 ## State
 
-### Procedures
+### Procedures and base types
 
 `Procedures` define the rule according to which votes are run. There can only 
 be one active procedure at any given time. If governance wants to change a 
 procedure, either to modify a value or add/remove a parameter, a new procedure 
 has to be created and the previous one rendered inactive.
 
+
 ```go
+type Procedure struct {
+  VotingPeriod      int64               //  Length of the voting period. Initial value: 2 weeks
+  MinDeposit        sdk.Coins           //  Minimum deposit for a proposal to enter voting period. 
+  Threshold         rational.Rational   //  Minimum propotion of Yes votes for proposal to pass. Initial value: 0.5
+  Veto              rational.Rational   //  Minimum value of Veto votes to Total votes ratio for proposal to be vetoed. Initial value: 1/3
+  MaxDepositPeriod  int64               //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
+  GovernancePenalty sdk.Rat               //  Penalty if validator does not vote
+  
+  IsActive          bool                //  If true, procedure is active. Only one procedure can have isActive true.
+}
+```
 
-type VoteType byte
+The current active procedure is stored in a global `params` KVStore.
+
+And some basic types:
+
+```go
+
+type Vote byte
 
 const (
-    VoteTypeYes         = 0x1
-    VoteTypeNo          = 0x2
-    VoteTypeNoWithVeto  = 0x3
-    VoteTypeAbstain     = 0x4
+    VoteYes         = 0x1
+    VoteNo          = 0x2
+    VoteNoWithVeto  = 0x3
+    VoteAbstain     = 0x4
 )
 
 type ProposalType  byte
 
 const (
-    ProposalTypePlainText = 0x1
+    ProposalTypePlainText       = 0x1
     ProposalTypeSoftwareUpgrade = 0x2
-
 )
 
-type Procedure struct {
-  VotingPeriod      int64               //  Length of the voting period. Initial value: 2 weeks
-  MinDeposit        int64               //  Minimum deposit for a proposal to enter voting period. 
-  VoteTypes         []VoteType          //  Vote types available to voters. 
-  ProposalTypes     []ProposalType      //  Proposal types available to submitters. 
-  Threshold         rational.Rational   //  Minimum propotion of Yes votes for proposal to pass. Initial value: 0.5
-  Veto              rational.Rational   //  Minimum value of Veto votes to Total votes ratio for proposal to be vetoed. Initial value: 1/3
-  MaxDepositPeriod  int64               //  Maximum period for Atom holders to deposit on a proposal. Initial value: 2 months
-  GovernancePenalty int64               //  Penalty if validator does not vote
-  
-  IsActive          bool                //  If true, procedure is active. Only one procedure can have isActive true.
-}
-```
+type ProposalStatus byte
 
-The current active procedure is stored in a global `params` KVStore.
+const (
+    ProposalStatusOpen      = 0x1   // Proposal is submitted. Participants can deposit on it but not vote
```

Comment:
is Open a clear enough term?

Reply:
I'm open to better alternatives. I think it's ok, but we can find better